src/utils/id.js

```diff
@@ -0,0 +1,21 @@
+const didJWT = require('did-jwt')
+const DID_MUPORT_PREFIX = 'did:muport:'
+
+module.exports = {
+  isMuportDID: (address) => address.startsWith(DID_MUPORT_PREFIX),
+  isClaim: async (claim, opts = {}) => {
+    try {
+      await didJWT.decodeJWT(claim, opts)
+      return true
+    } catch (e) {
+      return false
+    }
+  },
+  verifyClaim: async (claim, opts = {}) => {
+    try {
+      return await didJWT.verifyJWT(claim, opts)
+    } catch (e) {
+      return null
+    }
```

Comment:
Let's get rid of the try/catch and just return the result of `verifyJWT` directly!
Will be a pain to debug if invalid claims just return null regardless of the errors.